serving-demo/manifests/pipeline.yaml

```diff
@@ -20,70 +22,91 @@ spec:
             url: "redis://redis:6379"
 
     - name: planner
+      containerTemplate:
+        imagePullPolicy: Never
       scale:
         min: 1
       udf:
         container:
           image: ascii:0.1
+          imagePullPolicy: Never
           args:
             - planner
 
     - name: tiger
+      containerTemplate:
+        imagePullPolicy: Never
       scale:
         min: 1
       udf:
         container:
           image: ascii:0.1
+          imagePullPolicy: Never
           args:
             - tiger
 
     - name: dog
+      containerTemplate:
+        imagePullPolicy: Never
       scale:
         min: 1
       udf:
         container:
           image: ascii:0.1
+          imagePullPolicy: Never
           args:
             - dog
 
     - name: elephant
+      containerTemplate:
+        imagePullPolicy: Never
       scale:
         min: 1
       udf:
         container:
           image: ascii:0.1
+          imagePullPolicy: Never
           args:
             - elephant
 
     - name: asciiart
+      containerTemplate:
+        imagePullPolicy: Never
       scale:
         min: 1
       udf:
         container:
           image: ascii:0.1
+          imagePullPolicy: Never
           args:
             - asciiart
 
     - name: serve-sink
+      containerTemplate:
+        imagePullPolicy: Never
       scale:
         min: 1
       sink:
         udsink:
           container:
             image: servesink:0.1
+            imagePullPolicy: Never
             env:
               - name: NUMAFLOW_CALLBACK_URL_KEY
                 value: "X-Numaflow-Callback-Url"
               - name: NUMAFLOW_MSG_ID_HEADER_KEY
                 value: "X-Numaflow-Id"
 
     - name: error-sink
+      containerTemplate:
+        imagePullPolicy: Never
       scale:
         min: 1
       sink:
         udsink:
           container:
             image: servesink:0.1
+            imagePullPolicy: Never
```

Comment:
Should we set it to Always, since this is experimental?

Reply:
In this example we want to use the image we build locally. We don't want the image to be pulled from a container registry right?

Reply:
as per the readme, we are doing local import

Reply:
Yes. So it shouldn't be pulled from the container registry. Instead, use what's available locally